schemas/biomarkers.json

```diff
@@ -0,0 +1,481 @@
+      "name": "ca19-9_level",
+      "valueType": "integer",
+      "restrictions": {
+        "range": {
+           "exclusiveMin": 0
+        }
+      },
+      "meta": {
+        "displayName": "CA19-9 Level"
+      }
+    },
+    {
+      "description": "Indicate the quantitative measurement of the amount of CRP, an inflammatory marker, in the blood in mg/L. (Reference: NCIt C64548)",
+      "name": "crp_levels",
+      "valueType": "integer",
+      "restrictions": {
+        "range": {
+           "exclusiveMin": 0
+        }
+      },
+      "meta": {
+        "displayName": "C-reactive protein (CRP) Level"
+      }
+    },
+    {
+      "description": "Indicate the level of lactate dehydrogenase, in IU/L.",
+      "name": "ldl_level",
+      "valueType": "integer",
+      "restrictions": {
+        "range": {
+           "exclusiveMin": 0
+        }
+      },
+      "meta": {
+        "displayName": "Lactate Dehydrogenase (LDH) Level"
+      }
+    },
+    {
+      "description": "Indicate the value for a hemotology labratory test for the absolute number of neutrophil cells present in a sample of peripheral blood from a donor, in cells/uL. (Reference: caDSR CDE ID 2180198)",
+      "name": "anc",
+      "valueType": "integer",
+      "restrictions": {
+        "range": {
+           "exclusiveMin": 0
+        }
+      },
+      "meta": {
+        "displayName": "Absolute Neutrophil Count (ANC)"
+      }
+    },
+    {
+      "description": "Indicate the number of lymphocytes found in a given volume of blood, as cells/uL. (Reference: NCIt: C113237)",
+      "name": "alc",
+      "valueType": "integer",
+      "restrictions": {
+        "range": {
+           "exclusiveMin": 0
+        }
+      },
+      "meta": {
+        "displayName": "Absolute Lymphocyte Count (ALC)"
+      }
+    },
+    {
+      "description": "Indicate whether donor is a carrier of a mutation in a BRCA gene.",
+      "name": "brca_carrier",
+      "valueType": "string",
+      "restrictions": {
+        "codelist": [
+           "BRCA1",
+           "BRCA2",
+           "Both BRCA1 and BRCA2",
+           "No",
+           "Not applicable",
+           "Unknown"
+        ]
+      },
+      "meta": {
+        "displayName": "BRCA Carrier"
+      }
+    },
+    {
+      "description": "Indicate the expression of estrogen receptor (ER). (Reference: NAACCR 3827)",
+      "name": "er_status",
+      "valueType": "string",
+      "restrictions": {
+        "codelist": [
+           "Cannot be determined",
+           "Negative",
+           "Not applicable",
+           "Positive",
+           "Unknown"
+        ]
+      },
+      "meta": {
+        "displayName": "Estrogen Receptor Status"
+      }
+    },
+    {
+      "description": "Indicate the Allred score. The Allred score is based on the percentage of cells that stain positive by IHC for estrogen receptor (ER) and the intensity of that staining. (Reference: NAACCR: 3828, caDSR CDE ID 2725288)",
```

Comment:
for consistency with below:
` ... that stain positive by immunohistochemistry (IHC)...`

Reply:
Updated to include this